test/functional/helpers/createFixture/clientScripts.js

```diff
@@ -16,9 +16,14 @@ import readCache from "read-cache";
 import { ClientFunction } from "testcafe";
 
 const alloyEnv = process.env.ALLOY_ENV || "int";
-
+const alloyProdVersion = process.env.ALLOY_PROD_VERSION;
 // eslint-disable-next-line no-console
-console.log("ALLOY ENV:", alloyEnv);
+console.log(`ALLOY ENV: ${alloyEnv}`);
+
+if (alloyProdVersion) {
+  // eslint-disable-next-line no-console
+  console.log(`ALLOY PROD VERSION: ${alloyProdVersion}`);
+}
```

Comment:
Do you think we should throw an error If ALLOY_ENV is "prod" and alloyProdVersion is not specified? Or perhaps we can re-think these parameters. Having both ALLOY_ENV and ALLOY_PROD_VERSION seems redundant. Having the ALLOY_PROD_VERSION implies that we want ALLOY_ENV="prod".

Reply:
I kept the separate parameters because I think combining them would make it a bit messy, but I am now throwing an error if `ALLOY_ENV` is `prod` and `ALLOY_PROD_VERSION` is not specified.